style.css

```diff
@@ -478,3 +483,250 @@ form {
   color: white;
   border-color: #2230d2;
 }
+
+/* Desktop Version */
+@media (min-width: 768px) {
+  /* Navigation bar */
+  .main-nav {
+    height: 10vh;
+    padding: 0 10%;
+  }
+
+  .brand-logo {
+    margin: 0 16vw 0 0;
+  }
+
+  .nav-links {
+    display: flex;
+    gap: 2.22vw;
+    font-weight: 500;
+    font-size: 0.9em;
+    line-height: 20px;
+  }
+
+  .menu-icon {
+    display: none;
+  }
+
+  /* Headline */
+  .intro-section {
+    background-image: url(assets/background-headline.png);
+    background-size: contain;
+    background-position: center;
+    margin-top: 10vh;
+    padding: 16vw 18vw 12vw 16vw; /* Top, Right, Down, Left */
+    height: 916px; /* Adjust this value */
+    display: flex;
+    justify-content: center;
+  }
+
+  .intro-section h1 {
+    font-size: 48px;
+    font-weight: 700;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.37px;
+    color: #091e42;
+  }
+
+  .intro-section p {
+    max-width: 63.88vw;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 28px;
+  }
+
+  .connect-btn {
+    font-size: 16px;
+    margin-top: 2vh;
+  }
+
+  /* Projects Section */
+  .project-section {
+    padding: 10vw;
+    row-gap: 10vw;
+  }
+
+  .project-card {
+    display: flex;
+  }
+
+  .project-img {
+    align-self: center;
+  }
+
+  .project-img img {
+    width: calc(35vw - 1em);
+  }
+
+  .project-content {
+    margin: 0 3.2vw;
+  }
+
+  .project-content h2 {
+    padding-top: 12px;
+  }
+
+  .project-info {
+    padding-top: 12px;
+  }
+
+  .right {
+    flex-direction: row-reverse;
+  }
+
+  .right .project-content {
+    margin: 0 2.2vw;
+  }
+
+  .hide-on-mobile {
+    display: block;
+  }
+
+  /* About Section */
+  .about-section {
+    display: flex;
+    justify-content: start;
+    padding-top: 1vh;
+  }
+
+  .about-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: flex-start;
+  }
+
+  .about-description {
+    width: 447px;
+    height: 102px;
+  }
+
+  .about-info {
+    width: 65%;
+    min-width: 295px;
+    margin-left: 10vw;
+  }
+
+  .social-media-list {
+    display: flex;
+    list-style: none;
+    padding: 0;
+  }
+
+  .social-media-item {
+    margin-right: 0.6em;
+  }
+
+  .skills-section-wrapper {
+    width: 100vw; /* Set the width to 100% of the viewport width */
+    margin-right: 0; /* Remove the right margin */
+    padding-top: 1%;
+  }
+
+  .skills-section {
+    padding: 0;
+    position: relative;
+    width: 100%;
+    margin-left: 11vw;
+  }
```

Comment:
- [ ] Please extend the line under the about me tech categories section to full width to match closely with the provided figma design as according to the requirements you should stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).


|Figma design|Your design|
|---|----|
|<img width="587" alt="Screenshot 2023-06-16 at 11 04 53 AM" src="https://github.com/CVILLA09/Portfolio/assets/44798044/97a60cfc-840c-4b49-8525-9c808391af75">|<img width="787" alt="Screenshot 2023-06-16 at 11 21 43 AM" src="https://github.com/CVILLA09/Portfolio/assets/44798044/4c689e83-af1d-4fdf-a092-fa4f8bbe83a8">|

Suggestion:

Use `min-width:fit-content` and  `max-width:60%` on `.skills-section` to solve the issue. Example snippet:

```css
 .skills-section {
    padding: 0;
    position: relative;
    width: 100%;
    margin-left: 11vw;
    min-width: fit-content;
    max-width: 60%;
  }

```